docs/content.zh/docs/ops/traces.md

```diff
@@ -159,6 +159,14 @@ Flink reports a single span trace for the whole checkpoint and job initializatio
       <td>(Max/Sum)DownloadStateDurationMs<br><br>(optional - currently only supported by RocksDB Incremental)</td>
       <td>The aggregated (max and sum) across all subtasks duration of downloading state files from the DFS.</td>
     </tr>
+    <tr>
+      <td>(Max/Sum)RestoreStateSizeBytes.[location]<br><br>(optional - only reported if metrics.trace-restore-state-size is true)</td>
```

Comment:
Restore**d**StateSizeBytes ? (ditto in the actual metric name)

Reply:
Fine with me.